apps/frontend/src/app/modules/canteen/components/canteen/canteen.component.html

```diff
@@ -0,0 +1,19 @@
+<h1 class="text">Menza</h1>
+<div class="table">
+    <ng-container *ngFor="let week of [0, 1]">
+        <div class="menu-header">
+            <h2 class="text">{{week_prefixes[week]}}</h2>
```

Comment:
Also some formatting, please fix it everywhere
```suggestion
            <h2 class="text">{{ week_prefixes[week] }}</h2>
```

Reply:
Fixed by running the linter.